var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -103,6 +103,9 @@ class Mesa(MesonPackage):
     # OpenGL ES requires OpenGL
     conflicts('~opengl +opengles')
 
+    # https://gitlab.freedesktop.org/mesa/mesa/-/issues/5455
+    conflicts('llvm@13', when='@:21.3.1 +llvm')
```

Comment:
Better to use a compatibility bound instead of singling out one major version.

```suggestion
    conflicts('llvm@13.0.0:', when='@:21.3.1 +llvm')
```